atlasdb-config/src/test/resources/runtime-config-block.yml

```diff
@@ -26,3 +26,7 @@ sweep:
 
 internalSchema:
   targetTransactionsSchemaVersion: 1
+
+keyValueService:
+  type: cassandra
+  fetchReadLimitPerRow: 1000
```

Comment:
I think you need a new line at the end of the file, things seem to scream generally